lib/tasks/folio/developer_tools.rake

```diff
@@ -134,5 +134,25 @@ namespace :folio do
 
       puts "\n[folio:developer_tools:idp_correct_tiptap_node_attachments] Done."
     end
+
+    desc "Calculate and cache published usage counts for all files"
+    task calculate_file_published_usage_counts: :environment do
+      puts "Calculating published usage counts for all Folio::File records..."
+
+      total_count = Folio::File.count
+      processed = 0
+      batch_size = 500
+
+      Folio::File.find_in_batches(batch_size: batch_size) do |files|
+        files.each do |file|
```

Comment:
Myslím, že je na to ve Foliu či jinde nějaký progress tool, co vymalovává linku 0-100%